sklearn/metrics/_scorer.py

```diff
@@ -432,6 +432,7 @@ def check_scoring(estimator, scoring=None, *, allow_none=False):
         A string (see model evaluation documentation) or
         a scorer callable object / function with signature
         ``scorer(estimator, X, y)``.
+        If None, the provided estimator object's scoring method is used.
```

Comment:
It should probably be ``score`` method, with ticks, to be really explicit?

Reply:
that makes sense! I'll change it

Reply:
just done it @amueller . thanks for the feedback!